doc/api/next_api_changes/removals/22486-OG.rst

```diff
@@ -0,0 +1,7 @@
+Removal of deprecated methods and properties in ``lines`` and ``patches``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The deperecated properties ``validCap`` and ``validJoin``  have been removed
+from `~.Line2D` and `~.Patch` as the validation is centralized in ``rcsetup``.
+The deprecated methods ``get_dpi_cor`` and ``set_dpi_cor`` have been removed
+from `~.FancyArrowPatch` as the parameter ``dpi_cor`` is considered internal.
```

Comment:
`dpi_cor` is also deprecated from `FancyArrowPatch`, not internal. You should remove it also.

Reply:
Should I edit this as well then?

dpi_cor property of .FancyArrowPatch

This parameter is considered internal and deprecated.

From https://github.com/matplotlib/matplotlib/blob/main/doc/api/prev_api_changes/api_changes_3.4.0/deprecations.rst

Reply:
It was considered internal when it was deprecated; it will no longer exist when it is removed. There is no need to change historic changelogs.

Reply:
Ah, OK. I get it!

Reply:
I totally missed the parameter deletion (should use a better search expression...). But fixed now.